frontend/src/Editor/Inspector/EventManager.jsx

```diff
@@ -186,14 +207,24 @@ export const EventManager = ({
                     Alert Type
                   </div>
                   <div className="col-9" data-cy="alert-message-type">
-                    <SelectSearch
+                    <Select
                       className={`${darkMode ? 'select-search-dark' : 'select-search'}`}
                       options={alertOptions}
                       value={event.alertType}
                       search={false}
                       onChange={(value) => handlerChanged(index, 'alertType', value)}
-                      filterOptions={fuzzySearch}
                       placeholder="Select.."
+                      styles={{
```

Comment:
Same styles are being repeated at multiple places. Can we create a CSS class for this?